### ISSUE TITLE
bind_to_monitor works when you manually restart the app 

### ISSUE THREAD

**Issue by AmineDjeghri** [glzr-io/glazewm#503]
Edit : 
bind_to_monitor works only after you close and start the app again. Doesn't work when you do alt+shift+r

**Comment by AmineDjeghri** [comment]
Update : 
it works, but you need to manually restart the application. alt+shift+r doesn't do anything
I will edit the title

**Comment by lars-berger** [comment]
Closing in favor of #649
